README.md

```diff
@@ -69,9 +69,16 @@ To set up a development environment for the CLI
 
 1. Install go v1.17
 2. Install [buf](https://github.com/bufbuild/buf)
-3. make or make unit-tests to ensure everything built correctly.
+3. Install dependencies with `make dependencies` (`make dependencies -B` to force the target to run)
```

Comment:
Add that a `make all` so `dist` exists is needed.

Reply:
What I meant to say is that just `make bin` does not create the dist folder.

Reply:
README updated